### PR TITLE
Validated type=number in Modules

### PR DESCRIPTION
**Pull request by Shubhamverma2796**
### Summary of Changes

Validated type=number so that the user can not set negative values to follow proper form validation.

### Testing Instructions
1. Select Module-->Articles-Latest. Try to save the module with Articles to display = any -ve value. 
2. Select Module-->Articles - Newsflash. Try to save the module with Number of Articles = any -ve value. 
3. Select Module-->Articles - Related. Try to save the module with Maximum Articles = any -ve value. 
4. Select Module-->Modules: Banners. Try to save the module with Count = any -ve value. 
5. Select Module-->Feed Display. Try to save the module with Items to Display = any -ve value. 
6. Select Module-->Latest Users. Try to save the module with Number of Users = any -ve value.

### Actual result BEFORE applying this Pull Request

All the fields with the -ve values will be saved successfully before this pr.

### Expected result AFTER applying this Pull Request

Invalid field message is displayed.

### CODE REVIEW

File: modules/mod_users_latest/mod_users_latest.xml [hunk 1]
```diff
@@ -29,6 +29,8 @@
 					label="MOD_USERS_LATEST_FIELD_NUMBER_LABEL"
 					default="5"
 					filter="integer"
+					min="0"
```
Comment (Quy) [inline modules/mod_users_latest/mod_users_latest.xml]:
```suggestion
					min="1"
```

**Comment by pritam825** [comment]
I have tested this item :red_circle: unsuccessfully on e77961d24544cbb767cb2bed936b94357a53752c<br /><br />after applying the patch, it's not showing "Invalid field message is displayed".<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/36091">issues.joomla.org/tracker/joomla-cms/36091</a>.</sub>

**Comment by Shubhamverma2796** [comment]
> I have tested this item 🔴 unsuccessfully on [e77961d](https://github.com/joomla/joomla-cms/commit/e77961d24544cbb767cb2bed936b94357a53752c)after applying the patch, it's not showing "Invalid field message is displayed".
> 
> This comment was created with the [J!Tracker Application](https://github.com/joomla/jissues) at [issues.joomla.org/tracker/joomla-cms/36091](https://issues.joomla.org/tracker/joomla-cms/36091).

so form is getting submitted with -ve values?

**Comment by pritam825** [comment]
yes<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/36091">issues.joomla.org/tracker/joomla-cms/36091</a>.</sub>

**Comment by Ravi-chandra69** [comment]
I have tested this item :white_check_mark: successfully on e77961d24544cbb767cb2bed936b94357a53752c<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/36091">issues.joomla.org/tracker/joomla-cms/36091</a>.</sub>


Before applying patch:
Screenshot (291)

Result : Module saved successfully with a negative value.

After applying patch:

https://user-images.githubusercontent.com/94922241/143077149-62f70fdf-3244-4351-a102-89907c97b640.mp4

Result: Module not saved and "Invalid field" is displayed.

**Comment by pritam825** [comment]
@Ravi-chandra69 but it shouldn't be happen, according to the issue , after submitting with the negative value it should display invalid field<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/36091">issues.joomla.org/tracker/joomla-cms/36091</a>.</sub>

**Comment by Ravi-chandra69** [comment]
> @Ravi-chandra69 but it shouldn't be happen, according to the issue , after submitting with the negative value it should display invalid field
> This comment was created with the [J!Tracker Application](https://github.com/joomla/jissues) at [issues.joomla.org/tracker/joomla-cms/36091](https://issues.joomla.org/tracker/joomla-cms/36091).

I think you misunderstood the expected results. Before the patch, the module gets saved with the negative values also after clicking on the save button. But, after applying the patch, if the user tries to save the module with negative values, then the "Invalid field" message is displayed which is the expected behaviour.

You can see in the video uploaded by me, when I try to save with negative value then "invalid field" is displayed.

**Comment by pritam825** [comment]
I have tested this item :white_check_mark: successfully on e77961d24544cbb767cb2bed936b94357a53752c<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/36091">issues.joomla.org/tracker/joomla-cms/36091</a>.</sub>

**Comment by pritam825** [comment]
Oh ok Fine, Actually I was submitting direct from (http://localhost/joomla-4.0.5/administrator/index.php?option=com_cpanel&view=cpanel&dashboard=system) and through this it direct goes to main page without showing any error, so that's whit seems to me wrong, Now all fine, thanks @Ravi-chandra69 <hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/36091">issues.joomla.org/tracker/joomla-cms/36091</a>.</sub>

**Comment by Quy** [comment]
Thank you!

**Comment by Shubhamverma2796** [comment]
@pritam825  thanks for pointing out that the administration modules are still not getting validated. I will open a pr for the administrator module too.

**Pull request merged**